components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -96,13 +96,13 @@ const ContributorHighlightCard = ({ title, desc, prLink, user }: ContributorHigh
                   <span>Copy link</span>
                 </div>
               </DropdownMenuItem>
-              <DropdownMenuItem className="rounded-md">
+              <DropdownMenuItem className="rounded-md hidden">
```

Comment:
<img width="831" alt="Screenshot 2023-02-20 at 22 17 22" src="https://user-images.githubusercontent.com/237133/220191686-e534698c-0d15-4d2e-9bd5-d71a8005696d.png">

Reply:
something like this but for users to users?

Reply:
Not clear. Are you suggesting a follower endpoint similar to the stargaze one?

Reply:
nvm, this is for a reaction service. Looking at the code.

Reply:
no just trying to understand what "api updates" we are waiting on. these requirements should be mentioned in the design process, not when we are reviewing features to go into live testing environment :/

Reply:
I agree.